bloges56/DateSim
# Dialogue box change

## Description
- Dialogue scene now has 3 variables: "day" (only has test dialogue written for day 1 & 2), "relationshipProgress" (only has test dialogue for values 0 & 1), and "playerName" (can be changed to anything that can be contained in a string). The current dialogue yarn file should have a line or 2 for any day & relationship combo above, it also runs a test line for displaying the inputted player name and a test line for the function that changes the animation. 

## How To Test
1. Go to the Dialogue Scene
2. The DialogueSystem object has an attached DialogueHelper script with variables that can be changed within the inspector, the variable names are about the same as in the description above but with "edit" at the end 
3. Set the variables to the values you're testing before running 
4. Press Enter to progress dialogue, the box should just disappear when the current dialogue is done

## Checklist
- [x] Does the dialogue progress when Enter is pressed?
- [x] Does the "day" variable change the dialogue?
- [x] Does the "relationshipProgress" variable change the dialogue?
- [x] Does the "playerName" display the inputted name correctly?
- [x] Does the animation change for the last line and go back to the default/idle when the dialogue ends?
